cmd/puppet-db/import.go

```diff
@@ -1,21 +1,62 @@
 package main
 
 import (
+	"errors"
 	"fmt"
+	"os"
 
+	"github.com/puppetlabs/puppetdb-cli/api"
+	app "github.com/puppetlabs/puppetdb-cli/app/puppet-db"
+	"github.com/puppetlabs/puppetdb-cli/log"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 var importCmd = &cobra.Command{
 	Use:   "import",
 	Short: "import <path>",
-	Run:   executeImportCommand,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("Must specify a single file to import")
+		}
+		return nil
+	},
+	Run: executeImportCommand,
 }
 
 func init() {
 	rootCmd.AddCommand(importCmd)
 }
 
 func executeImportCommand(cmd *cobra.Command, args []string) {
-	fmt.Print("not implemented")
+	url := viper.GetStringSlice("urls")[0]
+	filePath := args[0]
+
+	if len(viper.GetStringSlice("urls")) > 1 {
+		log.Debug(fmt.Sprintf("Multiple URLs passed, will only use the first one (%s)", url))
+	}
+
+	puppetDb := app.NewWithConfig(
+		url,
+		viper.GetString("cacert"),
+		viper.GetString("cert"),
+		viper.GetString("key"),
+		viper.GetString("token"))
+
+	resp, err := puppetDb.PostImportFile(filePath)
+
+	if e, ok := err.(*api.ArgError); ok {
```

Comment:
We could hook `executeImportCommand` to `RunE` as `RunE` is handling errors by printing error, usage and exiting with 1.

Then we can just pass ArgError as return value in this specific case and return nil for success (`exit(1)` at line 55 should remain, as we don't what  usage printed in case of application error)